tensorflow/contrib/distributions/python/kernel_tests/half_normal_test.py

```diff
@@ -288,9 +289,10 @@ def testHalfNormalSampleMultiDimensional(self):
 
   def testNegativeSigmaFails(self):
     with self.cached_session():
-      halfnorm = hn_lib.HalfNormal(scale=[-5.], validate_args=True, name="G")
-      with self.assertRaisesOpError("Condition x > 0 did not hold"):
-        halfnorm.mean().eval()
+      with self.assertRaisesWithPredicateMatch(errors.InvalidArgumentError,
+                                               "Condition x > 0 did not hold"):
+        halfnorm = hn_lib.HalfNormal(scale=[-5.], validate_args=True, name="G")
```

Comment:
`Unused variable 'halfnorm' [unused-variable]`

Reply:
Changed `halfnorm` to `_`